### DISCUSSION TITLE
Prettified Code

### DISCUSSION THREAD

**Discussion by R4356th** [Glitchy-Scratch/glitchy-scratch.github.io#14] [Announcements]
Now that #13 has been merged, we are no longer using minified code! This has made the work of adding glitches way easier than before. It also looks like the merge gave us an performance upgrade and project thumbnails, names, news etc. are loading properly. We can easily edit text in all languages now!

**Comment by GrahamSH-LLK** [comment]
Awesome!

**Comment by R4356th** [comment]
Thanks to @GrahamSH-LLK, our code will now be automatically updated every time a commit has been pushed! 🎉
